<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatGui</class>
 <widget class="QDialog" name="chatGui">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>376</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:black;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QTextEdit" name="friendsDisplay">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:  rgb(80,80,80);
border-style: white inset;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTextEdit" name="outputDisplay">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: rgb(80,80,80);
color: white;

</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="inputDisplay">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>400</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: rgb(80,80,80);
color: white;
border: solid 2px red;
</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inputDisplay</tabstop>
  <tabstop>outputDisplay</tabstop>
  <tabstop>friendsDisplay</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
